schema.sql

```diff
@@ -0,0 +1,287 @@
+    reviewed_at TIMESTAMP NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (application_id) REFERENCES pot_application (id),
+    FOREIGN KEY (reviewer_id) REFERENCES account (id)
+  );
+
+-- Table pot_payout
+CREATE TABLE
+  pot_payout (
+    id SERIAL PRIMARY KEY,
+    recipient_id VARCHAR NOT NULL,
+    amount VARCHAR NOT NULL,
+    amount_usd DECIMAL(10, 2),
+    ft_id VARCHAR NOT NULL,
+    paid_at TIMESTAMP,
+    tx_hash VARCHAR,
+    FOREIGN KEY (recipient_id) REFERENCES account (id)
+  );
+
+-- Table pot_payout_challenge
+CREATE TABLE
+  pot_payout_challenge (
+    id SERIAL PRIMARY KEY,
+    challenger_id VARCHAR NOT NULL,
+    pot_id INT NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    message TEXT NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (challenger_id) REFERENCES account (id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id)
+  );
+
+-- Table pot_payout_challenge_admin_response
+CREATE TABLE
+  pot_payout_challenge_admin_response (
+    id SERIAL PRIMARY KEY,
+    challenge_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    message TEXT,
+    resolved BOOL NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (challenge_id) REFERENCES pot_payout_challenge (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+-- Table donation
+CREATE TABLE
+  donation (
+    id SERIAL PRIMARY KEY,
+    donor_id VARCHAR NOT NULL,
+    total_amount VARCHAR NOT NULL,
+    total_amount_usd DECIMAL(10, 2),
+    net_amount VARCHAR NOT NULL,
+    net_amount_usd DECIMAL(10, 2),
+    ft_id VARCHAR,
+    pot_id INT,
+    matching_pool BOOLEAN NOT NULL,
+    message TEXT,
+    donated_at TIMESTAMP NOT NULL,
+    recipient_id VARCHAR,
+    protocol_fee VARCHAR NOT NULL,
+    protocol_fee_usd DECIMAL(10, 2),
+    referrer_id VARCHAR,
+    referrer_fee VARCHAR,
+    referrer_fee_usd DECIMAL(10, 2),
+    chef_id VARCHAR,
+    chef_fee VARCHAR,
+    chef_fee_usd DECIMAL(10, 2),
+    tx_hash VARCHAR,
+    FOREIGN KEY (donor_id) REFERENCES account (id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (recipient_id) REFERENCES account (id)
+  );
+
+-- Table activity
+CREATE TABLE
+  activity (
+    id SERIAL PRIMARY KEY,
+    signer_id VARCHAR NOT NULL,
+    receiver_id VARCHAR NOT NULL,
+    timestamp TIMESTAMP NOT NULL,
+    action_result VARCHAR,
+    tx_hash VARCHAR,
+    type
+      ENUM(
+        'Donate_Direct',
+        'Donate_Pot_Public',
+        'Donate_Pot_Matching_Pool',
+        'Register',
+        'Deploy_Pot',
+        'Process_Payouts',
+        'Challenge_Payout',
+        'Submit_Application',
+        'Update_Pot_Config',
+        'Add_List_Admin',
+        'Remove_List_Admin'
+      ) NOT NULL
+  );
+
+CREATE TABLE
+  pot_admin (
+    pot_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_id, admin_id),
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
```

Comment:
move this up so it's under `pot` table definition

Reply:
i don't understand, why should this be moved to pots?, it's more like a many to many junction table, since an account can be admin on different pots, and pots can have many admins.

Reply:
No no I was just saying, all the pot-related tables should be grouped together for ease of reading & understanding the schema file